packages/vats/test/bootstrapTests/supports.js

```diff
@@ -197,29 +200,35 @@ export const makeWalletFactoryDriver = async (
      * @returns {Promise<void>}
      */
     executeOffer(offer) {
-      const offerCapData = marshaller.serialize({
-        method: 'executeOffer',
-        offer,
-      });
+      const offerCapData = marshaller.serialize(
+        harden({
```

Comment:
I'll use this example to clarify by earlier `harden` questions. Even if nothing changes to make this `harden` necessary, it should still be added since this is the object literal expression where the object is first created. These should generally be hardened, whether or not something downstream also hardens them. Though this is test code, for which we are much more lenient in our `harden` practices. Nevertheless, please keep this one even if it is unnecessary.